engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java

```diff
@@ -256,17 +256,12 @@ private Vector3i calcCameraCoordinatesInChunkUnits() {
      * Updates the currently visible chunks (in sight of the player).
      */
     @Override
-    public void updateAndQueueVisibleChunks() {
-        updateAndQueueVisibleChunks(true, true);
-    }
-
-    @Override
-    public int updateAndQueueVisibleChunks(boolean fillShadowRenderQueue, boolean processChunkUpdates) {
+    public int updateAndQueueVisibleChunks(boolean isFirstRenderingStageForCurrentFrame) {
         statDirtyChunks = 0;
         statVisibleChunks = 0;
         statIgnoredPhases = 0;
 
-        if (processChunkUpdates) {
+        if (isFirstRenderingStageForCurrentFrame) {
```

Comment:
It should be noticed that perhaps here some readability was lost as the two boolean variables previously present somewhat documented what the consequences of it being the first rendering stage in the frame, namely that chunks updates need to be processed and the shadowRenderQueue needs to be filled. I could reinstate those albeit I would keep the new method signature, passing a single isFirstRenderingStageForCurrentFrame boolean.